hadoop-ozone/dist/src/main/compose/ozonefs/docker-compose.yaml

```diff
@@ -49,21 +49,53 @@ services:
       environment:
          ENSURE_SCM_INITIALIZED: /data/metadata/scm/current/VERSION
       command: ["/opt/hadoop/bin/ozone","scm"]
-   hadoop3:
+   hadoop32:
```

Comment:
Can do, if you prefer it, but I can't see any advantage. It would duplicate the work to maintain these docker-compose file. What I would do instead of this (long-term) is to create a parameter (.env file) for hadoop version and parameterize the test. But it will be way more slower (instead of starting one ozone cluster we need to start it again and again for each version)

Reply:
One advantage i can think of is to have separate test suites for hadoop 2 and 3. With hadoop 2 and 3 being first class we can change there configs independently in future. However i am ok with this approch for current patch. We can split it later as desired.